src/GenerateModel.py

```diff
@@ -1,188 +1,10 @@
-from AutomaticScaling import*
-from Parameter import*
-from SetPath import*
+from Modifier import*
 
-Model = 'Dumas'
-Geometry = "Box"#Cylinder
-H=1.8
+H=1.70
 m=105
+Model = 'DeLeva'#
+Geometry = "Cylinder" #Cylinder
+FileName="HumanModelGenerate"
+Control="On"
```

Comment:
Could you add a comment explaining what is this variable (you could add a line also for the other variables above)